README.md

```diff
@@ -72,7 +72,8 @@ For either type, it can accept:
 - the `platform` parameter which determines whether the production or sandbox
   APNS environment is used.
   Valid values are 'production' or 'sandbox'. If not provided, 'production' is used.
-
+- the `push_type` parameter which determines what 'apns-push-type' header is sent to
+  APNS. If not provided, the header is not sent.
```

Comment:
```suggestion
  APNs. If not provided, the header is not sent.
```

Annoyingly Apple has decided to be very irritating with the official casing of `APNs`.